styles/variables-base.less

```diff
@@ -524,25 +524,25 @@
 @agate-mediacontrols-button-bg-height: inherit;
 @agate-mediacontrols-button-bg-width: inherit;
 @agate-mediacontrols-button-client-width: inherit;
-@agate-mediacontrols-button-height: inherit;
+@agate-mediacontrols-button-height: none;
```

Comment:
this is not a valid value.
if you want to keep the same height that comes from the Button component, you can use @agate-button-height